data/get_hymns.py

```diff
@@ -25,10 +26,36 @@
 url = f"{base_url}?{query_string}"
 
 hymns = pd.read_csv(url)
-hymns = hymns[['displayTitle', 'authors']]
+
+hymns = hymns[['displayTitle', 'authors', 'textAuthNumber']]
+
 hymns = hymns.rename(columns={'displayTitle': 'title'})
 
 hymns.replace({np.nan: None}, inplace=True)
-        
-# export to json
-hymns.to_json("hymns.json", orient="records", indent=2)
\ No newline at end of file
+
+# Retrieve texts
+texts_url = "https://hymnary.org/files/hymnary/other/texts.csv"
+texts = pd.read_csv(texts_url)
+
+merged = pd.merge(hymns, texts[['textAuthNumber', 'yearsWrote']], on='textAuthNumber', how='left')
```

Comment:
We could join directly to `hymns` here to avoid introducing another variable.
```suggestion
hymns = hymns.merge(texts[['textAuthNumber', 'yearsWrote']], on='textAuthNumber', how='left')
```